presto-main/src/test/java/com/facebook/presto/operator/scalar/TestDateTimeFunctionsBase.java

```diff
@@ -193,6 +195,9 @@ public void testFromUnixTime()
         dateTime = new DateTime(2001, 1, 22, 3, 4, 5, 888, DATE_TIME_ZONE);
         seconds = dateTime.getMillis() / 1000.0;
         assertFunction("from_unixtime(" + seconds + ")", TimestampType.TIMESTAMP, sqlTimestampOf(dateTime, session));
+
+        SqlTimestamp expected = sqlTimestampOf(2024, 1, 1, 23, 11, 49, 580, UTC, session.getTimeZoneKey(), session.toConnectorSession());
+        assertFunction("from_unixtime(1.7041507095805E9)", TimestampType.TIMESTAMP, expected);
```

Comment:
Would you confirm that this test fails without the change?

Reply:
Maybe adding a comment that this particular double was causing loss of precision in the function before the fix and hence we are testing it here.
In case anyone wondering why this number.